snorkeling_helper/label_functions/dependency_cluster/global_network_datafile_generator.py

```diff
@@ -0,0 +1,251 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.9.1+dev
```

Comment:
Why do you use a development version of jupytext? Just curious. The latest stable version, I think, it's 1.11.

Reply:
>Why do you use a development version of jupytext? Just curious. The latest stable version, I think, it's 1.11.

This is based on ignorance of jupytext updating their code LOL. I just didn't get a chance to update this package.